src/components/AvatarCropModal/AvatarCropModal.js

```diff
@@ -163,6 +164,15 @@ const AvatarCropModal = (props) => {
         translateY.value = clamp(offsetY, [maxOffsetY * -1, maxOffsetY]);
     }, [imageContainerSize, scale, clamp]);
 
+    /**
+     * Calculate newScale value
+     * @param {Number} newSliderValue
```

Comment:
```suggestion
    /**
     * @param {Number} newSliderValue
```
No need to include a description if it doesn't really add any value (and this just basically repeats the method name).